docs/section5/Rule5-48.md

```diff
@@ -15,37 +15,26 @@
 **Data Lookup:** None  
 **Function Call:** 
 
-  1. get_zone_conditioning_category()
-  2. match_data_element()
+  1. match_data_element()
 
 ## Rule Logic:  
 
-- Get zone conditioning category for B_RMR: `zcc_b = get_zone_conditioning_category(B_RMR)`
-
-- Get zone conditioning category for P_RMR: `zcc_p = get_zone_conditioning_category(P_RMR)`
-
 - For each building segment in the B_RMR: `for building_segment_b in B_RMR.building.building_segments:`
 
   - For each thermal block in building segment: `for thermal_block_b in building_segment_b.thermal_blocks:`
 
     - For each zone in thermal block: `for zone_b in thermal_block_b.zones:`
 
-      - Check if zone is unconditioned or unenclosed: `if zcc_b[zone_b] in ["UNENCLOSED", "UNCONDITIONED"]:`
-
-        - Get zone infiltration: `infiltration_b = zone_b.infiltration`
+      - Get zone infiltration: `infiltration_b = zone_b.infiltration`
 
-        - Get matching zone in P_RMR: `zone_p = match_data_element(P_RMR, Zones, zone_b.id)`
+      - Get matching zone in P_RMR: `zone_p = match_data_element(P_RMR, Zones, zone_b.id)`
 
-          - Get zone infiltration in P_RMR: `infiltration_p = zone_p.infiltration`
+        - Get zone infiltration in P_RMR: `infiltration_p = zone_p.infiltration`
 
-            **Rule Assertion:**  
+          **Rule Assertion:**  
 
-            - Case 1: For each unconditioned and unenclosed zone, if zone infiltration air leakage rate, schedule and method in B_RMR matches that in P_RMR: `if ( infiltration_b.air_leakage_rate == infiltration_p.air_leakage_rate ) AND ( infiltration_b.modeling_method == infiltration_p.modeling_method ) AND ( infiltration_b.multiplier_schedule == infiltration_p.multiplier_schedule ): PASS`  
+          - Case 1: For each unconditioned and unenclosed zone, if zone infiltration air leakage rate in B_RMR matches that in P_RMR: `if infiltration_b.air_leakage_rate == infiltration_p.air_leakage_rate: PASS`  
 
```

Comment:
That is correct when applied to unconditioned and unenclosed spaces

Reply:
@xingcx this looks good to me. i think we only need to update it to test the infiltration at wind pressure using the correct schema data elements.